### PR TITLE
Handle 0-dim tensors in HuggingFace wrapper

### PR DESCRIPTION
**Pull request by kvfairchild**
Undesirable behavior: Scoring a model on the `Wikitext-accuracy` benchmark results in `IndexError: index -1 is out of bounds for dimension 0 with size 0`.

Fix: if predicted token ID (`pred_id`) is of size 0, access value via `.item`

### CODE REVIEW

File: brainscore_language/model_helpers/huggingface.py [hunk 1]
```diff
@@ -209,7 +209,7 @@ def predict_next_word(self, base_output: CausalLMOutput):
         logits = base_output.logits
         pred_id = torch.argmax(logits, axis=2).squeeze()
         # Note that this is currently only predicting the next *token* which might not always be entire words.
-        last_model_token_inference = pred_id[-1].tolist()
+        last_model_token_inference = pred_id[-1].tolist() if len(pred_id.size()) > 0 else pred_id.item()
```
Comment (mschrimpf) [inline brainscore_language/model_helpers/huggingface.py:212]: could this not just be `pred_id.item()` in all cases?

Comment (kvfairchild) [reply brainscore_language/model_helpers/huggingface.py:212]: No, it fails when handed a >0-dim tensor, which occurs in other benchmarks.

**Review by mschrimpf — APPROVED** [review]
can we add a test to make sure we don't accidentally change this back?

**Comment by kvfairchild** [comment]
I intend to add it to the integration tests but have been waiting a long time for output from `score` with `distilgpt2` and `wikitextAccuracy`... looks like it's going to take a while still even on a VM with decent compute.

**Comment by mschrimpf** [comment]
Yes, running this on CPU will be slow. Maybe we can test a subset of wikitext rather than a full integration test?

**Pull request merged**